### ISSUE TITLE
Create ticket from e-mail

### ISSUE THREAD

**Issue by hassansalmane** [osTicket/osTicket#3269]
Hi,

I really hope someone answers this query of mine.

I set up osTicket (v1.9.14) and email is working fine.
OsTicket system is able to send email/alerts to clients and the admin after tickets are created.

first, I'm confused between piping and polling.

But now, I also want the functionality where the system should automatically create a ticket if my client sends an email to : support@partnet.ma
it should automatically create a ticket in the system and this client should also get a usual reply with the ticket number

so, basically I want both methods: ticket creation via web and ticket creation by sending email to support@partnet.ma

Kindly help me at the earliest.

Thanks a lot

HAssan SALMANE

**Comment by ntozier** [comment]
This is not a bug report.  You would be better suited to posting your question at the support forums.
http://osticket.com/forum

I would also recommend that you read the wiki for how to setup email polling/piping and cron.
http://osticket.com/wiki/Email_settings#POP3.2FIMAP_Polling
http://osticket.com/wiki/POP3/IMAP_Setting_Guide
